### ISSUE TITLE
sitewide, design system, components [KEYBOARD]: Focused item MUST be contained by focus outline, pagination Next &gt;

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#248]
@jenstrickland commented on [Tue Apr 21 2020](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8255)

# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team: VSA Public Websites; Project: Find Forms MVP
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When focus is on the last item in the pagination the `>` is outside of the focus outline, but should be within.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- [ ] As a keyboard user, I want to tab through the page and identify where focus is.
- [ ] Create and/or modify unit tests for this component to include an aXe scan

## Environment

* Operating System: all
* Browser: all
* Server destination: all

## Steps to Recreate

1. Enter https://www.va.gov/find-forms?q=medical in browser
2. Tab through the content, to the last item in the pagination
3. Verify that the `>` is not enclosed by the focus outline

## Screenshots or Trace Logs

![Screen Shot 2020-04-21 at 8 32 10 PM](https://user-images.githubusercontent.com/57469/79928109-f2bbef00-840f-11ea-8d0f-af76f3a78aad.png)

**Comment by jenstrickland** [comment]
Once this issue is resolved, please notify the VSA Public Websites team so they may audit their work to ensure they no longer have the issue. Thanks!

**Comment by 1Copenut** [comment]
Started updating CSS and Pagination component today. Worked with Chris V. to understand the publishing process and order of updates.

**Comment by 1Copenut** [comment]
7/29 - Have a fix in progress for the vets-website Pagination component. Ran into troubles with the Vets-api, then later content build repos, trying to troubleshoot now.

This fix will require 4 separate PRs in 4 repos, so I won't be merging it until next week.

**Comment by holdenhinkle** [comment]
PR submitted - https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/694
